### PR TITLE
Update return and throws javadoc for SyncSession::update

### PR DESCRIPTION
**Pull request by hakonhall**

### CODE REVIEW

**Comment by hakonhall** [comment]
FYI @jvenstad

File: documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java [hunk 1]
```diff
@@ -6,6 +6,7 @@
 import com.yahoo.document.DocumentPut;
 import com.yahoo.document.DocumentRemove;
 import com.yahoo.document.DocumentUpdate;
+import com.yahoo.document.TestAndSetCondition;
```
Comment (vekterli) [inline documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java:9]: Is it preferable to use an explicit import or to use a fully qualified class name in the JavaDoc instead?

**Review by vekterli — APPROVED** [review]
:+1: one question regarding additional import

File: documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java (hunk 1 above)
Comment (hakonhall) [reply documentapi/src/main/java/com/yahoo/documentapi/SyncSession.java:9]: I have never discussed this with anyone, but I prefer it this way.

**Pull request merged**